README-idoverridegroup.md

```diff
@@ -0,0 +1,200 @@
+Idoverridegroup module
+============
+
+Description
+-----------
+
+The idoverridegroup module allows to ensure presence and absence of idoverridegroups and idoverridegroup members.
+
+
+Use Cases
+---------
+
+With idoverridegroup it is possible to manage group attributes within ID views. These attributes are for example the group name or gid.
+
+
+Features
+--------
+
+* Idoverridegroup management
+
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.4.0 and up are supported by the ipaidoverridegroup module.
+
+
+Requirements
+------------
+
+**Controller**
+* Ansible version: 2.13
+
+**Node**
+* Supported FreeIPA version (see above)
+
+
+Usage
+=====
+
+Example inventory file
+
+```ini
+[ipaserver]
+ipaserver.test.local
+```
+
+
+Example playbook to make sure test group test_group is present in idview test_idview
+
+```yaml
+---
+- name: Playbook to make sure test group test_group is present in idview test_idview.
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverridegroup:
```

Comment:
All example playbook tasks lack the `name`.

Since we're using examples for the parameters (rather than usage scenarios), should we keep using playbooks or use only tasks? I mean, instead of "example playbook" use "example task in the README files.

This would reduce the size of the README files, removing lots of duplicate stuff (playbook readers, same name for playbook and tasks, ...), and then we could add an example scenario with an example playbook to achieve, at least, one of the use cases mentioned in the README, which might include other tasks with other modules (e.g. in the automount modules).

Of course, this is not something that we should change in this PR, as it should be a work for the whole collection.

Reply:
Fixed the missing names.